src/pubsub/mod_pubsub_db.erl

```diff
@@ -214,3 +220,8 @@ extra_debug_fun(Fun) ->
 %% Internal functions
 %%====================================================================
 
+sanitize_reason(Map) when is_map(Map) ->
+    Map;
+sanitize_reason(Other) ->
```

Comment:
For disabled backends `pool_not_started` is returned or similar, instead of map. This was a quick fix I needed for debugging but indeed maybe it can be done better.